dev-test/argocd-apps/mattermost-instance.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: mattermost-instance
+  namespace: openshift-gitops
+spec:
+  destination:
+    namespace: mattermost-instance
+    server: 'https://kubernetes.default.svc'
+  source:
+    chart: mattermost-instance
+    repoURL: https://stakater.github.io/stakater-charts
+    targetRevision: 0.1.4
+    helm:
+      releaseName: mattermost-instance
+      values: |
+        tenant:
+          database:
+            connectionString: AgCHEddZlZgRm2NF+AD4e9ZTR82kyVsNDFwApmZm5BqeGWpDYtK+DCi0a767agAjwRR3mdpxzvpQKgmjz4I9yFI1sLzjr1N8AlcPKDtKDt+iD7NOX/C7mALo3qXeuRbc/YQQyyCODVzo9ob/WMDD1szXOb0QXAhO51DxFQTK06ql/elnisSwh5kKjKBRZzAx7kMEVYJ+awqIEOtJs5p+l6xtpfE1DJHB6eNrHXnw/jecM87H7DXH26+4zdCNf4Bb7aZxHbnTVmj3PUgXOFz2j/+x84glUWNXnBKALSKzP/wA5OYAMl1AH+NcOqQX8S/UOg5POb3tCxa8+ac8IJw8lBxTtl6ipk+sD1rEdnDB894iTUWxhkEDUuzTBtDGxB85AgT7J/JFrcQbhdbyCnEV5vWs7La7eJcJT7siGPZS1tN3S/c9xtw9kdNOqEAzmUf4bmvN9mALgF6lbddyGSDBMu5TcvCxgntHYk4eg5NCZK8+tdgdHwjJdNqRs8J+PpfwbjYgY9IDZYymy+qG4yR4OUFfMCIjhWGn7qIIhunWe0NfkawESFKWQYIFf05oOkkWSlmWOZqDkr8Kkn83amFRA05Wev7DyRb4PjTKf5ejMdxCF/vE8QzuYk2ZDvod+xvRgx/Ma6qj/YSMNfLWMmD33vxbW1DNDPWVv0FNJDxwnUXM1ZTDTbZY13Vn+EsIj5aUMOo61Q+Lj4nOQ/EFa6m4ECgv8svWb+BTqSjx1ZVDoVKw4fjh83uyJF2YZzvDqJutaNHaVmuMgZaLe2UVIpzzdA+OorXMaifz4FbxD/eHDmVOYJjn10OfiLgcsrH3oVuWxVc8UgPC65Vhm4YXhAlOGF/yp7399nIuoiFRfl7RoFr6XXp8Tw==
+  project: default
```

Comment:
@faizanahmad055 we should put mattermost AppProject here

Reply:
Updated